config/crd/serving.kubeflow.org_trainedmodels.yaml

```diff
@@ -1,23 +1,13 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
how do you generate with v1 crd?

Reply:
Now controller-gen will generate v1 crd by default, unless "crdVersions=v1beta1" is added. I'm not sure in Makefile some places have "$(CRD_OPTIONS)" followed "$(CONTROLLER_GEN)" but some does not. I think we can add "$(CRD_OPTIONS)" for all the places?